plm_sale_fix/security/base_plm_security.xml

```diff
@@ -1,38 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-        <data>
+<odoo>
 
-        		<record id="report_sale_report_view" model="ir.model.access">
-		            <field name="name">PLM Sale Report View</field>
-		            <field name="model_id" ref="sale.model_sale_report"/>
-		            <field name="group_id" ref="plm.group_plm_view_user"/>
-		            <field name="perm_read" eval="True"/>
-		            <field name="perm_write" eval="False"/>
-		            <field name="perm_create" eval="False"/>
-		            <field name="perm_unlink" eval="False"/>
-        		</record>  
+    <record id="report_sale_report_view" model="ir.model.access">
+        <field name="name">PLM Sale Report View</field>
+        <field name="model_id" ref="sale.model_sale_report"/>
+        <field name="group_id" ref="plm.group_plm_view_user"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_write" eval="0"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="0"/>
+    </record>
 
-<!-- Integration User  -->
-                <record id="plm_sale_report_integration" model="ir.model.access">
-		            <field name="name">PLM Sale Report</field>
-		            <field name="model_id" ref="sale.model_sale_report"/>
-		            <field name="group_id" ref="plm.group_plm_integrationuser"/>
-		            <field name="perm_read" eval="True"/>
-		            <field name="perm_write" eval="False"/>
-		            <field name="perm_create" eval="False"/>
-		            <field name="perm_unlink" eval="False"/>
-        		</record>
+    <!-- Integration User  -->
+    <record id="plm_sale_report_integration" model="ir.model.access">
+        <field name="name">PLM Sale Report</field>
+        <field name="model_id" ref="sale.model_sale_report"/>
+        <field name="group_id" ref="plm.group_plm_integration_user"/>

```

Comment:
replace this with the old name please

Reply:
This conflict is solved in `plm` pull request.
i change the name to snake case there too, so, it is working very good.